src/main/Company.java

```diff
@@ -0,0 +1,71 @@
+
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Company  {
+
+
+    private  CEO ceo;
+    private List<Manager> managerList = new ArrayList<>();
+    private  String name;
+
+    public Company(String name, CEO ceo){
+        this.name = name;
+        this.ceo = ceo;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void display() {
+
+        System.out.println(this.getName()+ this.ceo.getName());
```

Comment:
- Tutaj też te this przy ceo jest niepotrzebne jak w paru innych miejscach ogólnie to jest tak trochę na wyczucie ale jak nie masz pewności czy ten this pasuje a nie potrzebujesz go to nie dawaj (dorzuciłem do doca z pytaniami cytat o this'ie).
- Dorzuciłbym jeszcze jakąś wiadomość "company name: " + ... bo w tym momencie bardzo się to zlewa ale to taka pierdoła.